vault/resource_pki_secret_backend_config_cluster.go

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package vault
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/hashicorp/terraform-provider-vault/internal/consts"
+	"github.com/hashicorp/terraform-provider-vault/internal/provider"
+	"github.com/hashicorp/terraform-provider-vault/util"
+)
+
+var pkiClusterFields = []string{
+	consts.FieldPath,
+	consts.FieldAIAPath,
+}
+
+func pkiSecretBackendConfigClusterResource() *schema.Resource {
+	return &schema.Resource{
+		Create: pkiSecretBackendConfigClusterCreateUpdate,
+		Read:   provider.ReadWrapper(pkiSecretBackendConfigClusterRead),
+		Update: pkiSecretBackendConfigClusterCreateUpdate,
+		Delete: pkiSecretBackendConfigClusterDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: func(_ context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				id := d.Id()
+				if id == "" {
+					return nil, fmt.Errorf("no path set for import, id=%q", id)
+				}
+
+				parts := strings.Split(util.NormalizeMountPath(id), "/")
+				if err := d.Set("backend", parts[0]); err != nil {
+					return nil, err
+				}
+
+				return []*schema.ResourceData{d}, nil
+			},
+		},
+
+		Schema: map[string]*schema.Schema{
+			consts.FieldBackend: {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The path of the PKI secret backend the resource belongs to.",
+			},
+			consts.FieldPath: {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Specifies the path to this performance replication cluster's API mount path.",
+				Elem:        &schema.Schema{Type: schema.TypeString},
```

Comment:
Do we need this since this is a TypeString?

Reply:
Indeed, it was not needed. Now removed.